tools/sync_test_server/Dockerfile

```diff
@@ -2,21 +2,21 @@ FROM ubuntu:16.04
 
 ARG ROS_DE_VERSION
 
-# Add realm repo
+# Install all normal dependencies
 RUN apt-get update -qq \
-    && apt-get install -y curl npm \
-    && curl -s https://packagecloud.io/install/repositories/realm/realm-beta/script.deb.sh | bash
-    #&& curl -s https://packagecloud.io/install/repositories/realm/realm/script.deb.sh | bash
-    #&& curl -s https://packagecloud.io/install/repositories/realm/realm-testing/script.deb.sh | bash
+    && apt-get install -y curl
+
+# Install node using nodesource to be able to install desired major version
+# Not doing this will cause apt-get to install from 4.*
+RUN curl -sL https://deb.nodesource.com/setup_6.x | bash - \
```

Comment:
Why not use the official node image as the base for this Dockerfile? Then you wouldn't have to worry about installing the right version and it would be faster to build the image.

Reply:
How can you do that?